### DISCUSSION TITLE
docker 里面的时间和实际时间差8个小时。。。困扰好久了

### DISCUSSION THREAD

**Discussion by Elvin10311** [monlor/docker-xiaoya#180] [Q&A] [answered]
docker 里面的时间和实际时间差8个小时。。。困扰好久了
如何操作能给个教程么？

**Comment by monlor** [comment]
#174

**Reply by Elvin10311** [reply]
对不不懂代码的小白来说，这等于天书一样，不知道该在哪里操作。。。
大神方便详细说下么？

**Reply by monlor** [reply, marked as answer]
升级新版本就好了

**Reply by Elvin10311** [reply]
我昨天刚安装的，时间也是错的 ....

**Reply by Elvin10311** [reply]
![1](https://github.com/monlor/docker-xiaoya/assets/19567916/be809fde-b3f7-43c3-aa62-558b18fd7583)

**Reply by hanlihanshaobo** [reply]
> ![1](https://github.com/monlor/docker-xiaoya/assets/19567916/be809fde-b3f7-43c3-aa62-558b18fd7583)
> 
> 

换成main标签的镜像，latest镜像没更新

**Reply by monlor** [reply]
记错了 时区这个昨天没发布，今天发布了

**Reply by Elvin10311** [reply]
感谢大佬，我去试试

**Comment by Elvin10311** [comment]
感谢大佬，更新完新版本时间已经好了
